<ui version="4.0" >
 <class>Tab</class>
 <widget class="QWidget" name="Tab" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <widget class="QxtGroupBox" name="qxtGroupBox" >
     <property name="toolTip" >
      <string/>
     </property>
     <property name="statusTip" >
      <string>QxtGroupBox</string>
     </property>
     <property name="whatsThis" >
      <string>QxtGroupBox</string>
     </property>
     <property name="title" >
      <string>GroupBox</string>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QxtSpanSlider" name="qxtSpanSliderHor" >
        <property name="toolTip" >
         <string/>
        </property>
        <property name="statusTip" >
         <string>QxtSpanSlider</string>
        </property>
        <property name="whatsThis" >
         <string>QxtSpanSlider</string>
        </property>
        <property name="singleStep" >
         <number>5</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition" >
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval" >
         <number>15</number>
        </property>
        <property name="lowerValue" >
         <number>0</number>
        </property>
        <property name="upperValue" >
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2" >
       <widget class="QxtStars" name="qxtStarsHor" >
        <property name="statusTip" >
         <string>QxtStars</string>
        </property>
        <property name="whatsThis" >
         <string>QxtStars</string>
        </property>
        <property name="maximum" >
         <number>10</number>
        </property>
        <property name="pageStep" >
         <number>3</number>
        </property>
        <property name="value" >
         <number>7</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="0" column="4" >
       <widget class="QxtSpanSlider" name="qxtSpanSliderVer" >
        <property name="toolTip" >
         <string/>
        </property>
        <property name="statusTip" >
         <string>QxtSpanSlider</string>
        </property>
        <property name="whatsThis" >
         <string>QxtSpanSlider</string>
        </property>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="invertedAppearance" >
         <bool>true</bool>
        </property>
        <property name="invertedControls" >
         <bool>true</bool>
        </property>
        <property name="lowerValue" >
         <number>0</number>
        </property>
        <property name="upperValue" >
         <number>99</number>
        </property>
       </widget>
      </item>
      <item rowspan="2" row="0" column="0" >
       <widget class="QxtStars" name="qxtStarsVer" >
        <property name="statusTip" >
         <string>QxtStars</string>
        </property>
        <property name="whatsThis" >
         <string>QxtStars</string>
        </property>
        <property name="maximum" >
         <number>3</number>
        </property>
        <property name="pageStep" >
         <number>3</number>
        </property>
        <property name="value" >
         <number>1</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QxtCheckComboBox" name="qxtCheckComboBox" >
        <property name="toolTip" >
         <string/>
        </property>
        <property name="statusTip" >
         <string>QxtCheckComboBox</string>
        </property>
        <property name="whatsThis" >
         <string>QxtCheckComboBox</string>
        </property>
        <property name="defaultText" >
         <string>...</string>
        </property>
        <property name="checkedItems" >
         <stringlist/>
        </property>
        <property name="checkMode" >
         <enum>QxtCheckComboBox::CheckWholeItem</enum>
        </property>
        <item>
         <property name="text" >
          <string>Curabitur</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>eleifend</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>purus</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>id</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>velit</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2" colspan="2" >
       <widget class="QxtStringSpinBox" name="qxtStringSpinBox" >
        <property name="toolTip" >
         <string/>
        </property>
        <property name="statusTip" >
         <string>QxtStringSpinBox</string>
        </property>
        <property name="whatsThis" >
         <string>QxtStringSpinBox</string>
        </property>
        <property name="strings" >
         <stringlist>
          <string>Curabitur</string>
          <string>eleifend</string>
          <string>purus</string>
          <string>id</string>
          <string>velit</string>
         </stringlist>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QToolBox" name="toolBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageList" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>112</width>
        <height>112</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>List</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QxtListWidget" name="qxtListWidget" >
         <property name="statusTip" >
          <string>QxtListWidget</string>
         </property>
         <property name="whatsThis" >
          <string>QxtListWidget</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageTree" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>397</width>
        <height>169</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Tree</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QxtTreeWidget" name="qxtTreeWidget" >
         <property name="toolTip" >
          <string/>
         </property>
         <property name="statusTip" >
          <string>QxtTreeWidget</string>
         </property>
         <property name="whatsThis" >
          <string>QxtTreeWidget</string>
         </property>
         <property name="columnCount" >
          <number>2</number>
         </property>
         <property name="decorationStyle" >
          <enum>Qxt::Buttonlike</enum>
         </property>
         <column>
          <property name="text" >
           <string>1</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>2</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageTable" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>112</width>
        <height>112</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Table</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QxtTableWidget" name="qxtTableWidget" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item rowspan="2" row="0" column="0" >
    <widget class="QxtLabel" name="qxtLabel" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>4</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Click me!</string>
     </property>
     <property name="statusTip" >
      <string>QxtLabel</string>
     </property>
     <property name="whatsThis" >
      <string>QxtLabel</string>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth" >
      <number>2</number>
     </property>
     <property name="midLineWidth" >
      <number>0</number>
     </property>
     <property name="text" >
      <string>Lorem ipsum dolor sit amet, consectetuer adipiscing elit.</string>
     </property>
     <property name="rotation" >
      <enum>Qxt::CounterClockwise</enum>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="0" column="2" >
    <widget class="QxtPushButton" name="qxtPushButton" >
     <property name="toolTip" >
      <string/>
     </property>
     <property name="statusTip" >
      <string>QxtPushButton</string>
     </property>
     <property name="whatsThis" >
      <string>QxtPushButton</string>
     </property>
     <property name="text" >
      <string>&lt;h1>Nam &lt;font color="red">dapibus&lt;/font> semper.&lt;/h1></string>
     </property>
     <property name="rotation" >
      <enum>Qxt::Clockwise</enum>
     </property>
     <property name="textFormat" >
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QxtStars</class>
   <extends>QWidget</extends>
   <header>QxtStars</header>
  </customwidget>
  <customwidget>
   <class>QxtTableWidget</class>
   <extends>QTableWidget</extends>
   <header>QxtTableWidget</header>
  </customwidget>
  <customwidget>
   <class>QxtTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>QxtTreeWidget</header>
  </customwidget>
  <customwidget>
   <class>QxtLabel</class>
   <extends>QFrame</extends>
   <header>QxtLabel</header>
  </customwidget>
  <customwidget>
   <class>QxtStringSpinBox</class>
   <extends>QSpinBox</extends>
   <header>QxtStringSpinBox</header>
  </customwidget>
  <customwidget>
   <class>QxtListWidget</class>
   <extends>QListWidget</extends>
   <header>QxtListWidget</header>
  </customwidget>
  <customwidget>
   <class>QxtGroupBox</class>
   <extends>QGroupBox</extends>
   <header>QxtGroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QxtCheckComboBox</class>
   <extends>QComboBox</extends>
   <header>QxtCheckComboBox</header>
  </customwidget>
  <customwidget>
   <class>QxtPushButton</class>
   <extends>QPushButton</extends>
   <header>QxtPushButton</header>
  </customwidget>
  <customwidget>
   <class>QxtSpanSlider</class>
   <extends>QSlider</extends>
   <header>QxtSpanSlider</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
